static/app/views/onboarding/targetedOnboarding/setupDocs.tsx

```diff
@@ -39,6 +39,18 @@ type Props = {
 
 function SetupDocs({organization, projects, search}: Props) {
   const api = useApi();
+  const [selectedProjects, setSelectedProjects] = useState<string[]>();
+  const selectedProjectsSet = new Set(selectedProjects);
+  useEffect(() => {
+    api
+      .requestPromise(`/organizations/${organization.slug}/client-state/onboarding/`)
+      .then((lastState: ClientState) => {
+        if (lastState.platforms) {
+          setSelectedProjects(Object.values(lastState.platforms));
```

Comment:
@Neo-Zhixing What happens if we call `setSelectedProjects` on a project that doesn't exist anymore?

Reply:
Should be fine now, the project will be there but disabled.